src/test/java/org/jabref/gui/entryeditor/EntryEditorTest.java

```diff
@@ -0,0 +1,66 @@
+package org.jabref.gui.entryeditor;
+
+import javafx.scene.Scene;
+import javafx.scene.control.Label;
+import javafx.scene.control.Tab;
+import javafx.scene.control.TabPane;
+import javafx.stage.Stage;
+
+import org.jabref.logic.bibtex.LatexFieldFormatterPreferences;
+import org.jabref.model.database.BibDatabaseContext;
+import org.jabref.model.entry.BibEntry;
+
+import org.fxmisc.richtext.CodeArea;
+import org.junit.Test;
+import org.testfx.framework.junit.ApplicationTest;
+
+public class EntryEditorTest extends ApplicationTest {
+
+    public Stage stage;
+    public Scene scene;
+    public CodeArea area;
+    TabPane pane;
+    SourceTab sourceTab;
+
+    @Override
+    public void start(Stage stage) throws Exception {
+        area = new CodeArea();
+        area.appendText("some example\n text to go here\n across a couple of \n lines....");
+        sourceTab = new SourceTab(new BibDatabaseContext(), new LatexFieldFormatterPreferences());
```

Comment:
The Argument CountingUndoManager is missing in the signature of this test. I have tried to add it by simply adding `new CountingUndoManager()`, but then the test fails. Can you have a look at it?